storage/statedb/cache.go

```diff
@@ -88,14 +89,17 @@ func (cacheType TrieNodeCacheType) ToValid() TrieNodeCacheType {
 // NewTrieNodeCache creates one type of any supported trie node caches.
 // NOTE: It returns (nil, nil) when the cache type is CacheTypeLocal and its size is set to zero.
 func NewTrieNodeCache(config *TrieNodeCacheConfig) (TrieNodeCache, error) {
+	if config == nil {
+		return nil, errNilTrieNodeCacheConfig
+	}
```

Comment:
How about adding a simple unit test for this newly added check logic?

Reply:
I added a test case enabling TestNewTrieNodeCache.